torch/nn/modules/container.py

```diff
@@ -637,7 +642,7 @@ def update(self, parameters: Mapping[str, 'Parameter']) -> None:
                     raise ValueError("ParameterDict update sequence element "
                                      "#" + str(j) + " has length " + str(len(p)) +
                                      "; 2 is required")
-                self[p[0]] = p[1]
+                self[p[0]] = p[1]  # type: ignore[assignment]
```

Comment:
Same here, brief comment `# parameters as length-2 list too cumbersome to type, see ModuleDict.update comment`